File: Lib/FPVR/Interfaces/ITravelInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace FPVR
{
    /// <summary>
    /// Provides orientation information a vehicles direction of travel.
    /// </summary>
    public interface ITravelInfo
    {
        /// <summary>
        /// Gets information about the vehicles direction of travel.
        /// </summary>
        /// <remarks>
        /// A <see cref="NavigationAttitude"/> instance with information about the vehicles direction of travel.
        /// </remarks>
        TravelAttitude TravelAttitude { get; }
    }
}

File: Android/FPVR/app/src/main/java/com/solersoft/fpvr/fpvrlib/ITravelInfo.java
package com.solersoft.fpvr.fpvrlib;

/**
 * Provides orientation information a vehicles direction of travel.
 */
public interface ITravelInfo extends IVehicleService
{
    /**
     * Gets information about the vehicles direction of travel.
     * A {@Link NavigationAttitude} instance with information about the vehicles direction of travel.
     */
    public TravelAttitude getTravelAttitude();
}